src/FileDownloader.ts

```diff
@@ -58,20 +58,20 @@ export default class FileDownloader implements IFileDownloader {
         const timeoutInMs = settings?.timeoutInMs ?? DefaultTimeoutInMs;
         const retries = settings?.retries ?? DefaultRetries;
         const retryDelayInMs = settings?.retryDelayInMs ?? DefaultRetryDelayInMs;
-        try{
```

Comment:
We removed the `try catch` from the first part of the code, but wouldn't it make sense to keep catching potential errors from it and log them as errors? Typically, it seems that `onDownloadProgressChange` could easily throw as this is provided by the consumer of the API.